### PR TITLE
edit sandbox and demo plum frontend url

### PR DESCRIPTION
**Pull request by NickAzureDevops**
### Jira link (https://tools.hmcts.net/jira/browse/DTSPO-16883)

change url for sandbox and demo temporary 

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by reespozzi — COMMENTED** [review]
Can you do something like plum-test.env... instead? Not sure changing capital will be picked up

**Comment by NickAzureDevops** [comment]
This settings right ? Require branches to be up to date before merging @reespozzi

**Comment by NickAzureDevops** [comment]
settings have been disabled

File: apps/cnp/plum-frontend/demo.yaml [hunk 1]
```diff
@@ -8,4 +8,4 @@ spec:
     nodejs:
       # Testing spot instances
       replicas: 4
-      ingressHost: plum.Demo.platform.hmcts.net
+      ingressHost: plum-test.env.demo.platform.hmcts.net
```
Comment (reespozzi) [inline apps/cnp/plum-frontend/demo.yaml:11]:
```suggestion
      ingressHost: plum-test.demo.platform.hmcts.net
```

File: apps/cnp/plum-frontend/sbox.yaml [hunk 2]
```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
   values:
     nodejs:
-      ingressHost: plum.sbox.platform.hmcts.net
+      ingressHost: plum-test.env.sandbox.platform.hmcts.net
```
Comment (reespozzi) [inline apps/cnp/plum-frontend/sbox.yaml:8]:
```suggestion
      ingressHost: plum-test.sandbox.platform.hmcts.net
```

**Review by reespozzi — APPROVED** [review]

**Pull request merged**